codeserver/ubi9-python-3.12/nginx/root/usr/share/container-scripts/nginx/common.sh

```
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# get_matched_files finds file for image extending
+function get_matched_files() {
+  local custom_dir default_dir
+  custom_dir="$1"
+  default_dir="$2"
+  files_matched="$3"
+  find "$default_dir" -maxdepth 1 -type f -name "$files_matched" -printf "%f\n"
+  [ -d "$custom_dir" ] && find "$custom_dir" -maxdepth 1 -type f -name "$files_matched" -printf "%f\n"
+}
```

Shellcheck (0.10.0)

[warning] 'function' keyword is non-standard. Delete it.

(SC2112)